### PR TITLE
frontend docs refresh

### PR DESCRIPTION
**Pull request by krpeacock**
Changes around local network, blockchain and so on. 

I've also refreshed package versions to their current state and made some editorial improvements to a couple of the pages, based on my own judgment as the inheritor of these tutorials

The CSS tutorial had become redundant due to my improvements to the starter project, so I adapted it as a guide on how to import styles into JS, as well as configuring sass, a popular superset of the CSS language

### CODE REVIEW

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 1]
```diff
@@ -245,31 +245,31 @@ For example:
...
-Before you can build the `+custom_greeting+` project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+Before you can build the `+custom_greeting+` project, you need to connect to either the live {platform}, or a canister execution environment running locally in your development environment.
 
-To start the network locally:
+To start the environment locally:
 
 . Open a new terminal window or tab on your local computer.
 . Navigate to the root directory for your project, if necessary.
-. Start the {IC} network on your local computer by running the following command:
+. Start the {IC} environment on your local computer by running the following command:
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:256]:
```suggestion
. Start the local canister execution environment on your local computer by running the following command:
```
There is only one IC :)

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 2]
```diff
@@ -245,31 +245,31 @@ For example:
...
 +
 [source,bash]
 ----
 dfx start --background
 ----
 +
-After the local {IC} network completes its startup operations, you can continue to the next step.
+After the local {IC} environment completes its startup operations, you can continue to the next step.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:263]: dito

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 3]
```diff
@@ -245,31 +245,31 @@ For example:
...
 
-== Register, build, and deploy the application
+== Register, build, and deploy the dapp
 
-After you connect to the {IC} network running locally in your development environment, you can register, build, and deploy your application locally.
+After you connect to the local {IC} execution environment, you can register, build, and deploy your dapp locally.
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:267]:
```suggestion
After you connect to the local canister execution environment, you can register, build, and deploy your dapp locally.
```

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 4]
```diff
@@ -326,17 +326,17 @@ For example:
 +
 image:modified-result.png[Modified greeting result]
 
-== Stop the local network
+== Stop the local execution environment
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:329]:
```suggestion
== Stop the local canister execution environment
```

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc [hunk 5]
```diff
@@ -326,17 +326,17 @@ For example:
...
+After you finish experimenting with the front-end for your dapp, you can stop the local canister execution environment so that it doesn’t continue running in the background.
 
 To stop the local network:
 
 . In the terminal that displays the webpack development server, press Control-C to interrupt the dev-server.
 
 . In the terminal that displays network operations, press Control-C to interrupt the local network process.
 
-. Stop the {IC} network by running the following command:
+. Stop the {IC} environment by running the following command:
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/custom-frontend.adoc:339]:
```suggestion
. Stop the local canister execution environment by running the following command:
```

File: modules/developers-guide/pages/tutorials/my-contacts.adoc [hunk 6]
```diff
@@ -211,29 +239,29 @@ cd ../../..
...
-Before you can build the `+contacts+` project, you need to connect to the {IC} network either running locally in your development environment or running remotely on a subnet that you can access.
+Before you can build the `+contacts+` project, you need to connect to the local canister execution environment.
 
-To start the network locally:
+To start the environment locally:
 
 [arabic]
 . Open a new terminal window or tab on your local computer.
-. Start the {IC} network on your local computer by running the following command:
+. Start the canister execution environment on your local computer by running the following command:
```
Comment (yvonneanne) [inline modules/developers-guide/pages/tutorials/my-contacts.adoc:248]:
```suggestion
. Start the local canister execution environment on your local computer by running the following command:
```

**Review by yvonneanne — APPROVED** [review]

File: modules/developers-guide/pages/tutorials/custom-frontend.adoc (hunk 2 above)
Comment (krpeacock) [reply modules/developers-guide/pages/tutorials/custom-frontend.adoc:263]:
```suggestion
After the local canister execution environment completes its startup operations, you can continue to the next step.
```

**Pull request merged**